CHANGELOG.unreleased.md

```diff
@@ -1365,6 +1365,14 @@ const bigints = [200000n, 0x0000000an, 0b01111111n];
 const bigints = [200_000n, 0x0000_000an, 0b0111_1111n];
 ```
 
+#### VS Code: add support for .mongo files ([#6848] by [@aymericbouzy])
+
+When using the Azure Cosmos DB extension for VS Code, you can create .mongo files to write MongoDB queries, which use Javascript syntax. This change allows VS Code to format your file using Prettier.
+
+```js
+db.users.find({ someField: { $exists: true } });
+```
```

Comment:
Can you format changelog item as we do in others?

Reply:
I don't think this makes sense in this case.

Reply:
Is that what you mean?

``````md
<!-- prettier-ignore -->
```js
// Input
db.users.find({someField:{$exists: true}});

// Output
db.users.find({ someField: { $exists: true } });
```
``````

Reply:
yep, but maybe we really can simplify it here as you do

Reply:
I'm not making any change unless you tell me to 😄 I believe too it's not really necessary in this case.